### PR TITLE
fix: holdback percentage not allowing decimals

### PR DESCRIPTION
**Pull request by sam-warren**
Card: https://github.com/bcgov/cas-cif/issues/1698

### CODE REVIEW

File: app/data/jsonSchemaForm/fundingParameterEPUiSchema.ts [hunk 1]
```diff
@@ -34,6 +34,7 @@ export const fundingParameterEPUiSchema = {
   holdbackPercentage: {
     "ui:widget": "NumberWidget",
     isPercentage: true,
+    numberOfDecimalPlaces: 2,
```
Comment (BCerki) [inline app/data/jsonSchemaForm/fundingParameterEPUiSchema.ts:37]: Is it confirmed that it should be 2 places?

**Review by BCerki — COMMENTED** [review]
Nice! I'd suggest adding decimals to the mock data in the jest funding form test and summary tests, and one of the cypress tests, to make sure this displays as expected.

File: app/data/jsonSchemaForm/fundingParameterEPUiSchema.ts (hunk 1 above)
Comment (sam-warren) [reply app/data/jsonSchemaForm/fundingParameterEPUiSchema.ts:37]: Added two decimal places per the comment by @Sepehr-Sobhani on the ticket - @nanyangpro can you confirm what the expected number of decimal places are?

Comment (sam-warren) [reply app/data/jsonSchemaForm/fundingParameterEPUiSchema.ts:37]: Update: Confirmed with Nan that 2 decimal places is acceptable.

**Review by BCerki — APPROVED** [review]

**Pull request merged**